src/CoreBundle/Resources/public/javascript/jquery-form-4.2.2/jquery.form.min.js

```diff
@@ -0,0 +1,23 @@
+/*!
+ * jQuery Form Plugin
```

Comment:
We can't get rid of jquery-migrate anytime soon i think, because of BC breaks. I would not remove it before 8.0. But the jquery.form plugin shows no problem in the console. the jquery.migrate log is empty. So maybe this info is old.